internal/api/v2/projects_related_resources.go

```diff
@@ -0,0 +1,271 @@
+package api
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+
+	"github.com/hashicorp-forge/hermes/internal/server"
+	"github.com/hashicorp-forge/hermes/pkg/document"
+	"github.com/hashicorp-forge/hermes/pkg/models"
+	"gorm.io/gorm"
+)
+
+type ProjectRelatedResourcesGetResponse struct {
+	ExternalLinks   []ProjectRelatedResourcesGetResponseExternalLink   `json:"externalLinks,omitempty"`
+	HermesDocuments []ProjectRelatedResourcesGetResponseHermesDocument `json:"hermesDocuments,omitempty"`
+}
+
+type ProjectRelatedResourcesGetResponseExternalLink struct {
+	Name      string `json:"name"`
+	URL       string `json:"url"`
+	SortOrder int    `json:"sortOrder"`
+}
+
+type ProjectRelatedResourcesGetResponseHermesDocument struct {
+	GoogleFileID   string `json:"googleFileID"`
+	Title          string `json:"title"`
+	DocumentType   string `json:"documentType"`
+	DocumentNumber string `json:"documentNumber"`
+	SortOrder      int    `json:"sortOrder"`
+}
+
+type ProjectRelatedResourcesPutRequest struct {
+	ExternalLinks   []ProjectRelatedResourcesPutRequestExternalLink   `json:"externalLinks,omitempty"`
+	HermesDocuments []ProjectRelatedResourcesPutRequestHermesDocument `json:"hermesDocuments,omitempty"`
+}
+
+type ProjectRelatedResourcesPutRequestExternalLink struct {
+	Name      string `json:"name"`
+	URL       string `json:"url"`
+	SortOrder int    `json:"sortOrder"`
+}
+
+type ProjectRelatedResourcesPutRequestHermesDocument struct {
+	GoogleFileID string `json:"googleFileID"`
+	SortOrder    int    `json:"sortOrder"`
+}
+
+func projectsResourceRelatedResourcesHandler(
+	srv server.Server,
+	w http.ResponseWriter,
+	r *http.Request,
+	projectID uint,
+) {
+	logArgs := []any{
+		"path", r.URL.Path,
+		"project_id", projectID,
+	}
+
+	switch r.Method {
+	case "GET":
+		logArgs = append(logArgs, "method", r.Method)
+
+		// Get project.
+		proj := models.Project{}
+		if err := proj.Get(srv.DB, projectID); err != nil {
+			if errors.Is(err, gorm.ErrRecordNotFound) {
+				srv.Logger.Warn("project not found", logArgs...)
+				http.Error(w, "Project not found", http.StatusNotFound)
+				return
+			} else {
+				srv.Logger.Error("error getting project from database",
+					append([]interface{}{
+						"error", err,
+					}, logArgs...)...)
+				http.Error(
+					w, "Error processing requst", http.StatusInternalServerError)
```

Comment:
```suggestion
					w, "Error processing request", http.StatusInternalServerError)
```

Reply:
Good catch, I really leaned into this typo haha 🙈